### PR TITLE
Anon contact export import 2

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/contacts/models.py [hunk 1]
```diff
@@ -767,7 +771,7 @@ def prepare_fields(cls, field_dict, import_params=None, user=None):
         return field_dict
 
     @classmethod
-    def get_import_file_headers(cls, csv_file):
+    def get_org_import_file_headers(cls, csv_file, org):
```
Comment (norkans7) [inline temba/contacts/models.py]: added org in the args to know when we have anon orgs

**Comment by norkans7** [comment]
- removes urn fields on export for anon orgs
- allow import of files missings urns on anon org if there is a uuid field
- ignore columns with empty header for all orgs
- tests

Please review

File: temba/contacts/tests.py [hunk 2]
```diff
@@ -1619,18 +1621,44 @@ def test_from_urn(self):
...
+        with self.assertRaises(Exception):
+            Contact.validate_org_import_header(['name'], self.org)  # missing a URN
 
         with self.assertRaises(Exception):
-            Contact.validate_import_header(['name'])  # missing a URN
+            Contact.validate_org_import_header(['phone', 'twitter', 'external'], self.org)  # missing name
 
         with self.assertRaises(Exception):
-            Contact.validate_import_header(['phone', 'twitter', 'external'])  # missing name
+            Contact.validate_org_import_header(['uuid', 'name'], self.org)
```
Comment (nicpottier) [inline temba/contacts/tests.py]: I think this should be valid in the non-anon case as well.

Comment (norkans7) [inline temba/contacts/tests.py]: ok

**Comment by nicpottier** [comment]
Looks good apart from the import of uuid,name for non-anon orgs

File: temba/contacts/tests.py (hunk 2 above)
Comment (norkans7) [inline temba/contacts/tests.py]: is it required actually to have the name column if we have the uuid one?

**Comment by norkans7** [comment]
One thing here is that we create a new contact without urn if the uuid is inexistent and set the other fields

File: temba/contacts/tests.py (hunk 2 above)
Comment (nicpottier) [inline temba/contacts/tests.py]: I think we require name everywhere else no?

Comment (norkans7) [inline temba/contacts/tests.py]: yep, I was thinking of just allowing update by uuid for field without requiring the name column

Comment (norkans7) [inline temba/contacts/tests.py]: if we have uuid we just check there is at least one other field even if it is not a name

Comment (nicpottier) [inline temba/contacts/tests.py]: Ya, I'm fine with that.

**Pull request merged**